packages/typespec-test/test/openai_modular/generated/typespec-ts/src/api/models.ts

```diff
@@ -96,7 +98,7 @@ export interface CompletionsLogProbabilityModel {
 }
 
 /** Representation of the manner in which a completions response concluded. */
-/** "stopped", "tokenLimitReached", "contentFiltered" */
+/** "stop", "length", "content_filter" */
```

Comment:
feel like the original enum maybe make more sense ? 
https://github.com/Azure/autorest.typescript/pull/1904/files#diff-bf776f1db792b622f5ae2e56967f5295fac95afd9968bf7c708bb7c3a96c5d1dR22-R37